src/explore-education-statistics-common/src/modules/table-tool/contexts/TableHeadersContext.tsx

```diff
@@ -0,0 +1,109 @@
+import useToggle from '@common/hooks/useToggle';
+import noop from 'lodash/noop';
+import React, {
+  createContext,
+  ReactNode,
+  useContext,
+  useMemo,
+  useState,
+} from 'react';
+
+export interface TableHeadersContextState {
+  activeList?: string;
+  defaultNumberOfItems: number;
```

Comment:
Don't think we need this a context value. It creates an unnecessary dependency on the context and it'd be simpler for this to be controlled the components themselves via props.